src/commands/config.command.ts

```diff
@@ -48,19 +43,53 @@ export function registerConfigCmd(program: Command) {
       "Disconnect from Google Cloud Platform",
       `${CLICommand} config --disconnect GCP`,
     )
-    .action(async (options: CmdConfigOpts) => {
-      await config(options, configCmd);
-    });
+    .action(configurePlatforms);
+
+  program.command("config", connectCmd);
+
+  const listCmd = new Command()
+    .description(
+      "Show config.",
+    )
+    .action(showConfig);
+
+  const setupAzureManagementGroupCmd = new Command()
+    .arguments("<management_group_id:string>")
+    .description(
+      "Setup a parent Management Group in Azure for all your Subscriptions to query. This will speed up price collection because an optimized query can be used.",
+    )
+    .example(
+      "Set a parent management group ID",
+      `${CLICommand} config azure managementgroup set 4a2ef91d-7697-4759-ab36-0f8049d274df`,
+    )
+    .action(setupAzureManagementGroup);
+
+  const azureSubCmd = new Command().description(
+    "Configure Azure related options",
+  ).command("managementgroup", setupAzureManagementGroupCmd);
 
-  program
-    .command("config", configCmd.command("list", listCmd));
+  connectCmd
+    .command("list", listCmd)
+    .command("azure", azureSubCmd);
 }
 
-function config(options: CmdConfigOpts, program: Command) {
+function setupAzureManagementGroup(
+  options: CmdConfigOpts,
+  managementGroupId: string,
+) {
+  setupLogger(options);
+
+  const config = loadConfig();
+  config.azure.parentManagementGroups = [managementGroupId];
+  writeConfig(config);
+  log.info(`Set Azure root management group ID to: ${managementGroupId}`);
+}
+
+function configurePlatforms(options: CmdConfigOpts, program: Command) {
   setupLogger(options);
 
   if (Object.keys(options).length === 0) {
-    log.info(`Please see "${CLICommand} config -h" for more information.`);
+    log.info(`Please see ${CLICommand} config -h for more information.`);
```

Comment:
I would keep the "" so it's easier to see what exactly the command is.

Reply:
hm interesting. I think auto format removed this...